database/entity/0053-change_password_encryption/User.ts

```diff
@@ -34,21 +34,6 @@ export class User extends BaseEntity {
   })
   alias: string
 
-  @Column({ name: 'public_key', type: 'binary', length: 32, default: null, nullable: true })
```

Comment:
make a migration and a new file defining the whole new database model - if you need help how this works please contact me. You can never edit an existing migration. I has already ran on some machines and therefore you need to make a new migration.

Reply:
oh that is true, sorry